includes/internal/student-progress/quiz-progress/repositories/class-comments-based-quiz-progress-repository.php

```diff
@@ -160,6 +160,7 @@ public function delete_for_quiz( int $quiz_id ): void {
 		];
 		$comments      = Sensei_Utils::sensei_check_for_activity( $activity_args, true );
 		foreach ( $comments as $comment ) {
+			$this->delete_grade_and_answers( $comment->comment_ID );
```

Comment:
Does this need testing?

Reply:
It has a test already, and I had to return this call because the test started failing :)